suite-common/wallet-core/src/device/deviceUtils.ts

```diff
@@ -17,3 +17,17 @@ export const DeviceError = (message: string): DeviceError => ({
     type: 'DeviceError' as const,
     message,
 });
+
+// Check if error is an entropy check failure, but not one of the temporarily skipped errors (to be investigated)
+export const isIgnoredEntropyCheckError = (error: string) => {
```

Comment:
Since it's a function, I'd prefer to reflect that in the name.
```suggestion
export const getIsIgnoredEntropyCheckError = (error: string) => {
```

Reply:
Good idea, done